go-controller/pkg/node/user_defined_node_network_controller.go

```diff
@@ -16,29 +16,29 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 )
 
-// SecondaryNodeNetworkController structure is the object which holds the controls for starting
-// and reacting upon the watched resources (e.g. pods, endpoints) for secondary network
-type SecondaryNodeNetworkController struct {
+// UserDefinedNodeNetworkController structure is the object which holds the controls for starting
+// and reacting upon the watched resources (e.g. pods, endpoints) for User Defined Networks
+type UserDefinedNodeNetworkController struct {
```

Comment:
shouldn't it be UserDefinedNetworkNodeController? now it sounds like users defined nodes

Reply:
I just followed the previous wording. If you make it UserDefinedNetworkNodeController, I guess you could argue it now looks like a Node Controller